DYDX-Bots/Large_USDC_Staking-Withdrawals/src/agent.spec.ts

```diff
@@ -0,0 +1,324 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { encodeParameter, encodeParameters } from "forta-agent-tools";
+import { createAddress, TestTransactionEvent, MockEthersProvider } from "forta-agent-tools/lib/tests";
+import { BigNumber } from "ethers";
+import { provideHandleTransaction } from "./agent";
+import { STAKED_SIG, WITHDREW_STAKE_SIG, WITHDREW_DEBT_SIG, USDC_IFACE } from "./utils";
+import BalanceFetcher from "./balance.fetcher";
+
+const testModuleUsdcBalance: BigNumber = BigNumber.from("10000000000000000000"); // 10
+const testThresholdPercentage: number = 20;
+const testStaker: string = createAddress("0xac");
+const testBlockNumbers: number[] = [2, 42, 92, 360, 444, 3500, 90210, 972011, 3524233];
+const testAmounts: BigNumber[] = [
+  BigNumber.from("5000000000000000000"), // 5
+  BigNumber.from("6000000000000000000"), // 6
+  BigNumber.from("7000000000000000000"), // 7
+  BigNumber.from("8000000000000000000"), // 8
+  BigNumber.from("9000000000000000000"), // 9
+];
+const testLowAmounts: BigNumber[] = [
+  BigNumber.from("1000000000000000"), // .001
+  BigNumber.from("2000000000000000"), // .002
+  BigNumber.from("3000000000000000"), // .003
+  BigNumber.from("4000000000000000"), // .004
+];
+
+describe("Large Stake Token Deposit/Withdrawal Test Suite", () => {
```

Comment:
Running the test passes with the following console log: 
`BigNumber.toString does not accept any parameters; base-10 is assumed`. The code generating that message from ethers.js library can be found here https://github.com/ethers-io/ethers.js/blob/fc1e006575d59792fa97b4efb9ea2f8cca1944cf/packages/bignumber/src.ts/bignumber.ts#L203.
It confused me why it is detecting an argument being passed to `toString()`, while there is none.